go.sum

```diff
@@ -28,143 +28,28 @@ cloud.google.com/go v0.90.0/go.mod h1:kRX0mNRHe0e2rC6oNakvwQqzyDmg57xJ+SZU1eT2aD
 cloud.google.com/go v0.93.3/go.mod h1:8utlLll2EF5XMAV15woO4lSbWQlk8rer9aLOfLh7+YI=
```

Comment:
all the changes to the go.sum / go.work.sum files look spurious... you should be able to revert these to match master:

* `go.sum`
* `go.work.sum`
* `staging/src/k8s.io/code-generator/examples/go.work.sum`
* `staging/src/k8s.io/kms/internal/plugins/_mock/go.work.sum`

Reply:
Thank you! Reverted